courseme/util/json.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
Having the file encoding set is good practice.  Try creating a file with a string literal containing a GBP sign to see why...

```python
s = "£10"
```